extensions/amp-audio/1.0/test/validator-amp4ads-amp-audio.html

```diff
@@ -0,0 +1,49 @@
+<!--
```

Comment:
please remove `amp4ads` input and output files as these should not be enabled for the format by default